cddl/conditional-reference-series-triple-record.cddl

```diff
@@ -1,19 +1,18 @@
 $$triples-map-extension //= (
-  &(conditional-reference-series-triples: 8) => [ + conditional-reference-series-triple-record ]
+  &(conditional-reference-series-triples: 8) => 
+    [ + conditional-reference-series-triple-record ]
 )
-  
-; The triple subject is a compound structure consisting of an environment identifier plus measurements that, when matched to evidence, is a condition of the object.
-; The object is a time series of conditional reference and endorsed triples
 
 conditional-reference-series-triple-record = [
-  [ 
-    environment-map,
-    measurement-map / measurement-values-map
-  ] ; triple subject is a compound statement
-  [ + reference-endorsed-record ] ; triple object is a series
+  ; triple subject
+  [ environment-map,
+    measurement-map / measurement-values-map ; initial reference values
+  ] 
+  ; triple object
+  [ + reference-endorsed-record ]
 ]
  
 reference-endorsed-record = [
-  measurement-map / measurement-values-map  ; reference values
-  ? measurement-map / measurement-values-map  ; conditionally endorsed values, could be empty, but still a valid expression in a series
+    measurement-map / measurement-values-map ; addit'l reference values
+  ? measurement-map / measurement-values-map ; endorsed values
```

Comment:
is there a syntactical bug with measurement-map / measurement-values-map?

Reply:
Note that tooling doesn't complain about syntax and examples pass schema checks. If there is a problem, it doesn't appear to be syntactical.